apps/meeting/src/components/DeviceSelection/CameraDevices/VideoTransformDropdown.tsx

```diff
@@ -0,0 +1,101 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  Device,
+  isVideoTransformDevice,
+  VideoTransformDevice,
+} from 'amazon-chime-sdk-js';
+import React, { ChangeEvent, useState, useEffect } from 'react';
+import { useBackgroundBlur } from 'amazon-chime-sdk-component-library-react';
+import { useBackgroundReplacement } from 'amazon-chime-sdk-component-library-react';
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import { FormField } from 'amazon-chime-sdk-component-library-react';
+import { Select } from 'amazon-chime-sdk-component-library-react';
+import { VIDEO_TRANSFORM_OPTIONS } from '../../../types/index';
+
+interface Props {
+  /* Title for the dropdown */
+  label?: string;
+}
+
+export const VideoTransformDropdown: React.FC<Props> = ({
+  label = 'Video Transform Dropdown',
+}) => {
+  const [transformOption, setTransformOption] = useState(VIDEO_TRANSFORM_OPTIONS.none);
+  const {isBackgroundBlurSupported,createBackgroundBlurDevice} =  
```

Comment:
Perhaps some comments here explaining why we have both hooks here for BB and BR, since we dont have documentation on these components.

Reply:
done